index.html

```diff
@@ -0,0 +1,671 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New York Times</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav class="left-nav">
+            <img src="./img/menu.png" alt="menu" />
+            <img src="./img/search.png" alt="search" />
+            <a class="space-cosmos">Space & Cosmos</a>
+        </nav>
+        <nav>
+            <img src="./img/logo.PNG" alt="logo" />
+        </nav>
```

Comment:
[REQUIRED] Please add <a> tag on the logo image. It should be link also not only an image.